website_react/src/components/dataCollectionForms/dataAccessForm.jsx

```diff
@@ -0,0 +1,147 @@
+import InputString from '../inputs/InputString'
+import InputInteger from '../inputs/InputInteger'
+import InputBoolean from '../inputs/InputBoolean'
+import InputENUM from '../inputs/InputENUM'
+import InputList from '../inputs/InputList'
+
+
+const DataAccessForm = () => {
```

Comment:
In this implementation, we are letting the View (React) manage our data.
we can hold this fields array for each class we are falling and reuse a single component called `Form`
also a code duplication `DataAccessForm`, `DBConnectionForm`, `DBEntityForm`, and so on and so fourth 
What do you thnk about crating a single component called `Form` and reuse it with different data each time?

Reply:
let's talk about it.